client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -14,7 +14,7 @@ import NewVersionInfoView from './NewVersionInfoView';
 
 import UpdateChecksAPI from './UpdateChecksAPI';
 
-import Flags, { DISABLE_SERVER_INTERACTION } from '../../util/Flags';
+import Flags, { DISABLE_SERVER_INTERACTION, FORCE_UPDATE_CHECKS } from '../../util/Flags';
```

Comment:
As discussed: Please move the configuration consern into one place (UpdateChecks) and pass the configured endpoint downstream.